charts/consent/values.yaml

```diff
@@ -100,7 +100,7 @@ proxyLogLevel:
 proxyImageRepository:
 proxyImageVersion:
 
-replicas: 1
+replicas: 3
```

Comment:
Does this cause any downtime? Mostly curious as I don't know much about how replicas are handled.